### PR TITLE
docs: add trezor support content

### PR DESCRIPTION
**Pull request by agraebe**
fixing #782

### CODE REVIEW

File: src/pages/stacks-wallet/usage.md [hunk 1]
```diff
@@ -285,3 +285,17 @@ you do not restart the wallet, you can simple close it.
...
    The wallet displays the **Terms of Use**. You can accept to enter the wallet
    again or choose **Quit**.
+
+## Trezor wallet support
+
+Due to technical compatibility issues, we are currently unable to fully support all Trezor wallets. Going forward, support for any Trezor wallet will be discontinued.
+
+### Trezor One and current Stacks Wallet
+
+The current Stacks Wallet (v3) supports only the Trezor One device, and only up to firmware [version 1.6.1](https://wiki.trezor.io/Firmware_changelog). If you have a newer firmware, you will have to downgrade by [following instructions described on this page](https://wiki.trezor.io/Firmware_downgrade).
```
Comment (yknl) [inline src/pages/stacks-wallet/usage.md:295]: Trezor firmware up to v1.8.3 are supported

Comment (agraebe) [reply src/pages/stacks-wallet/usage.md]: thanks, updated. have another look, please. especially, the ledger suggestion?

File: src/pages/stacks-wallet/usage.md [hunk 2]
```diff
@@ -285,3 +285,17 @@ you do not restart the wallet, you can simple close it.
...
+
+### Trezor One and current Stacks Wallet
+
+The current Stacks Wallet ([v3.0.0](https://github.com/blockstack/stacks-wallet/releases/tag/v3.0.0)) supports only the Trezor One device, and only up to firmware [version 1.8.3](https://wiki.trezor.io/Firmware_changelog). If you have a newer firmware, you will have to downgrade by [following instructions described on this page](https://wiki.trezor.io/Firmware_downgrade).
+
+!> Downgrading your wallet is dangerous and will erase your Trezor, so make sure you have your seed phrase backed up.
+
+### Coming Stacks Wallet
+
+With the release of the new Stacks Wallet (v4), coming out after Stacks 2.0, we will remove support for _any_ Trezor wallet. It is therefore recommended to use your Stacks (STX) tokens with a [Ledger hardware device](https://www.ledger.com/). You can enter your seed phrase into the Ledger device to access your account.
```
Comment (yknl) [inline src/pages/stacks-wallet/usage.md:301]: Note that only Ledger Nano S is currently supported by v3.0 of the Stacks wallet. However all versions should be supported after Stacks 2.0.

Comment (agraebe) [reply src/pages/stacks-wallet/usage.md:303]: cool, thanks!

**Review by yknl — APPROVED** [review]

File: src/pages/stacks-wallet/usage.md [hunk 3]
```diff
@@ -285,3 +285,21 @@ you do not restart the wallet, you can simple close it.
 
    The wallet displays the **Terms of Use**. You can accept to enter the wallet
    again or choose **Quit**.
+
+## Trezor wallet support
+
+Due to technical compatibility issues, we are currently unable to fully support all Trezor wallets. Going forward, support for any Trezor wallet will be discontinued.
```
Comment (markmhx) [inline src/pages/stacks-wallet/usage.md:291]: While we expect Trezor support to get discontinued upon mainnet, it's not an absolutely 100% done deal. There's a small chance we we do support it upon or soon after mainnet. So we may not want to provide such forward guidance quite yet.

File: src/pages/stacks-wallet/usage.md [hunk 4]
```diff
@@ -285,3 +285,21 @@ you do not restart the wallet, you can simple close it.
...
+
+### Trezor One and current Stacks Wallet
+
+The current Stacks Wallet ([v3.0.0](https://github.com/blockstack/stacks-wallet/releases/tag/v3.0.0)) supports only the Trezor One device, and only up to firmware [version 1.8.3](https://wiki.trezor.io/Firmware_changelog). If you have a newer firmware, you will have to downgrade by [following instructions described on this page](https://wiki.trezor.io/Firmware_downgrade).
+
+!> Downgrading your wallet is dangerous and will erase your Trezor, so make sure you have your seed phrase backed up.
+
+### Coming Stacks Wallet
+
+With the release of the new Stacks Wallet (v4), coming out after Stacks 2.0, we will remove support for _any_ Trezor wallet. It is therefore recommended to use your Stacks (STX) tokens with a [Ledger hardware device](https://www.ledger.com/). You can enter your seed phrase into the Ledger device to access your account.
```
Comment (markmhx) [inline src/pages/stacks-wallet/usage.md:303]: Ditto here. I'd recommend we don't provide forward guidance about the new wallet just yet.

**Review by markmhx — COMMENTED** [review]
I suggest modifying this new content to remove any forward-looking statements about the new wallet since things on that front are still TBD.

Didn't we want to include instructions for migrating from Trezor to Ledger as well here to see if that gets traction as an alternative option to downgrading the firmware?

**Pull request merged**